src/main/java/com/intershop/oms/test/servicehandler/omsdb/v1/OMSDbHandlerV1.java

```diff
@@ -222,12 +222,55 @@ public Long runDBStmtLongById(String query, Long id, String resultColumnName)
     }
 
     @Override
-    public ArrayList<String> runDBStmtString(String query, String resultColumnName)
+    public List<Long> runDBStmtLongListById(String query, Long id, String resultColumnName)
```

Comment:
optional parameters should better be in the last positions

Reply:
Nothing optional here?

Reply:
I meant nullable parameter.

Reply:
I just wanted to be nice! ;-) (and accept nulls as well)

Reply:
But the method is called "...ById" and that what it really for....